packages/react-slider/src/components/Slider/Spec.md

```diff
@@ -1,146 +1,42 @@
-| showValue         | Whether to show the value on the right of the Slider.                                                                                                  | Renamed to showValueLabel                                                            |
-| onChanged         | Callback on mouse up, touch end, or after key presses have stopped.                                                                                    | Replaced with onChange                                                               |
-| snapToStep        | Whether to decide that thumb will snap to closest value while moving the slider                                                                        | The control should keep a consistent visual behavior.                                |
-| buttonProps       | Additional props for the actual `role="slider"` (slider box) element. (Note that this element is not actually a button in the current implementation.) | Unsure if it is an essential API feature.                                            |
-| valueFormat       | Custom formatter for the slider value. value                                                                                                           | Label will handle custom formats                                                     |
-| originFromZero    | Whether to attach the origin of slider to zero. Helpful when the range include negatives. @defaultvalue false                                          | Replaced with origin to allow for more control over the component.                   |
-| ranged            | Whether to render a **Ranged Slider**. Ranged Sliders display two `thumbs` that allow for lower and upper bounds to be easily selected.                | Ranged will be separated into a different component.                                 |
-| label             | The description label of the **Slider**.                                                                                                               | Can be handled by a form component.                                                  |
-
 ## Structure
 
 - _**Public**_
   ```jsx
-  <Slider defaultValue={3} marks={marks} />
+  <Slider defaultValue={50} min={10} max={100} step={5} />
   ```
 - _**Internal**_
 
   ```jsx
   <slots.root {...slotProps.root}>
+    <slots.input {...slotProps.input} />
     <slots.rail {...slotProps.rail} />
-    <slots.track {...slotProps.track} />
     <slots.thumb {...slotProps.thumb} />
-    <slots.input {...slotProps.input} />
   </slots.root>
   ```
 
 - _**DOM** - how the component will be rendered as HTML elements_
 
 ```jsx
 <div className="fui-Slider">
-  <div className="fui-Slider-rail" />
-  <div className="fui-Slider-track" />
-  <div className="fui-Slider-thumb" />
-  <div className="fui-Slider-input" />
+  <div className="fui-Slider__input" />
```

Comment:
the input slot tagName is `input`, not `div` -- this seems maybe outdated

Reply:
```suggestion
  <input className="fui-Slider__input" />
```